src/scripting/MatchFunctions.js

```diff
@@ -81,8 +81,12 @@ const equals = (ignoreCase) => (botresponse, utterance) => {
 }
 
 const wer = () => (botresponse, utterance, args) => {
-  botresponse = _normalize(botresponse || '')
-  utterance = toString(utterance || '')
+  const _prepareString = str => {
```

Comment:
can you please make this a shared function, because we will shortly add a new flag to ignore punctuation mark generally, and we will need this then.